Modules/Loadable/Markups/SubjectHierarchyPlugins/qSlicerSubjectHierarchyMarkupsPlugin.cxx

```diff
@@ -139,6 +143,10 @@ void qSlicerSubjectHierarchyMarkupsPluginPrivate::init()
   this->ToggleSelectPointAction->setObjectName("ToggleSelectPointAction");
   QObject::connect(this->ToggleSelectPointAction, SIGNAL(triggered()), q, SLOT(toggleSelectPoint()));
 
+  this->SetTerminologyAction = new QAction("Set markup terminology", q);
+  this->SetTerminologyAction->setObjectName("SetTerminologyAction");
```

Comment:
To be consistent with DeleteNodeAction name, either add "Node" to the action name to make it clear that it is for the entire node, not for a point (we may use terminology in the future to specify what a markup point is); or remove "Node" from both actions.

To be consistent with "EditPropertiesAction" name, I think it would be better to replace "Set" by "Edit".